src/main/java/com/google/api/codegen/util/ruby/RubyCommentReformatter.java

```diff
@@ -17,17 +17,44 @@
 import com.google.api.codegen.CommentPatterns;
 import com.google.api.codegen.util.CommentReformatter;
 import com.google.common.base.Splitter;
+import com.google.common.base.Strings;
 import java.util.regex.Matcher;
 
 public class RubyCommentReformatter implements CommentReformatter {
   @Override
   public String reformat(String comment) {
```

Comment:
I can see that, but for readability, the list reformatting logic would ideally be in a discrete code unit, unless there's something that prevents that.